packages/amplify_authenticator/lib/src/blocs/auth/auth_bloc.dart

```diff
@@ -228,6 +235,32 @@ class StateMachineBloc {
     }
   }
 
+  Stream<AuthState> _checkUserVerification() async* {
+    try {
+      List<String> unverifiedAttributeKeys =
+          await _authService.getUnverifiedAttributeKeys();
+      if (unverifiedAttributeKeys.isNotEmpty) {
+        _authEventController.add(
+          AuthSetUnverifiedAttributeKeys(
+            AuthSetUnverifiedAttributeKeysData(
+              unverifiedAttributeKeys: unverifiedAttributeKeys,
+            ),
+          ),
+        );
+        yield AuthFlow(screen: AuthScreen.verifyUser);
+      } else {
+        yield const Authenticated();
+      }
+    } on Exception catch (e) {
+      // TODO: How should exceptions be handled in this flow
```

Comment:
It seems like the exceptions that would come back from this would not be useful for an end user, and should only happen if there is some sort of internal issue with our code (or maybe with a call to Cognito for attributes, if that's what's happening here)?

If that's the case, maybe we should just log these exceptions but not display them view the ExceptionController?